code/web/src/setup/helpers.js

```diff
@@ -71,8 +71,8 @@ export function formatSurvey(products) {
 export function styleToString(style) {
   const key = {
     1: 'Casual',
-    2: 'Edgy',
-    3: 'Classy'
+    2: 'Classy',
+    3: 'Edgy'
```

Comment:
I was wondering about this, but I wasn't sure. I guess I'm actually 'Classy' after all.